src/lib/y2users/linux/writer.rb

```diff
@@ -101,8 +101,15 @@ def initialize(config, initial_config)
       def write
         issues = Y2Issues::List.new
 
+        # handle groups first as it does not depend on users uid, but it is vice versa
+        new_groups = config.groups.map(&:id) - initial_config.groups.map(&:id)
+        new_groups.map! { |id| config.groups.find { |g| g.id == id } }
+        new_groups.each { |g| add_group(g, issues) }
+
+        # TODO: modify group?
```

Comment:
Note: maybe we could include a `GroupFinder` similar to `UserFinder`

Reply:
well, we agreed on introduction that collections, so I do not want to do unnecessary work when it is planned to be replaced.

Reply:
Sorry, I didn't know / remind it :innocent: